packages/components/src/composite/stories/index.story.tsx

```diff
@@ -353,3 +353,28 @@ const Fill = ( { children } ) => {
 		},
 	},
 };
+
+export const RemoveActiveItem: StoryObj< typeof Composite > = {
```

Comment:
Was going to say this story doesn't make much sense to keep, but I see you created it solely for testing and plan to remove it - thanks!